src/libraries/System.Private.Xml/src/System/Xml/XmlQualifiedName.cs

```diff
@@ -135,18 +135,16 @@ public static string ToString(string name, string ns)
         }
 
         // --------- Some useful internal stuff -----------------
-        internal void Init(string name, string ns)
+        internal void Init(string? name, string? ns)
         {
-            Debug.Assert(name != null && ns != null);
-            _name = name;
-            _ns = ns;
+            _name = name ?? string.Empty;
```

Comment:
Why the change of contract here?

Reply:
I changed it since all constructors accepted null and converted that to non-null - I think one of the callsites was passing nullable to Init which would make the `_name` nullable which would pretty much always throw NRE on all methods so instead of that I decided to just copy behavior from the constructor